lib/core/presentation/pages/home/views/widgets/home_event_card/widgets/home_event_card_footer_left.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:app/core/application/auth/auth_bloc.dart';
+import 'package:app/core/domain/event/entities/event.dart';
+import 'package:app/core/presentation/widgets/theme_svg_icon_widget.dart';
+import 'package:app/core/presentation/pages/home/views/widgets/home_event_card/widgets/home_event_card_price_info.dart';
+import 'package:app/core/utils/event_utils.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/theme/color.dart';
+import 'package:app/theme/sizing.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class HomeEventCardFooterLeft extends StatelessWidget {
```

Comment:
you already create home_event_card folder, then should move these mini widget into that folders, otherwise the folders looks messy

Reply:
sr, nhìn lộn :D